apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -233,6 +233,15 @@ internal SpannerConnectionStringBuilder CloneWithNewDataSource(string dataSource
                 DataSource = dataSource
             };
 
+        /// <summary>
+        /// Returns a new instance of a <see cref="SpannerConnectionStringBuilder"/> with the database
+        /// portion of the DataSource replaced with a new value.
+        /// </summary>
+        /// <param name="database">the new database name.</param>
```

Comment:
Can this be null or empty?
(Nit: capital letter to start both this and the returns part)

Reply:
+1  can be null and should switch to a different string if so.